napari_segment_blobs_and_things_with_membranes/__init__.py

```diff
@@ -944,3 +987,139 @@ def sub_sample(image:"napari.types.ImageData", sample_x: int = 1, sample_y: int
 @time_slicer
 def squeeze(image:"napari.types.ImageData") -> "napari.types.ImageData":
     return np.squeeze(image)
+
+
+@register_function(menu="Segmentation post-processing > Grayscale erosion (scikit-image, nsbatwm)")
+@jupyter_displayable_output(library_name='nsbatwm', help_url='https://www.napari-hub.org/plugins/napari-segment-blobs-and-things-with-membranes')
+@time_slicer
+def grayscale_erosion(labels: "napari.types.LabelsData", radius: int = 1,
+                      footprint_shape: FootprintShape = FootprintShape.square) -> "napari.types.LabelsData":
+    """
+    Applies grayscale erosion to an image, using a footprint with the given radius and shape.
+
+    See also
+    --------
+    ..[0] https://scikit-image.org/docs/stable/api/skimage.morphology.html#skimage.morphology.erosion
+    """
+
+    footprint = _generate_footprint(footprint_shape, radius, labels.ndim)
+    return erosion(labels, footprint=footprint)
+
+
+@register_function(menu="Segmentation post-processing > Binary erosion (scikit-image, nsbatwm)")
+@jupyter_displayable_output(library_name='nsbatwm', help_url='https://www.napari-hub.org/plugins/napari-segment-blobs-and-things-with-membranes')
+@time_slicer
+def binary_erosion(labels: "napari.types.LabelsData", radius: int = 1,
+                   footprint_shape: FootprintShape = FootprintShape.square) -> "napari.types.LabelsData":
+    """
+    Applies binary erosion to an image, using a footprint with the given radius and shape.
+
+    See also
+    --------
+    ..[0] https://scikit-image.org/docs/stable/api/skimage.morphology.html#skimage.morphology.binary_erosion
+    """
+
+    footprint = _generate_footprint(footprint_shape, radius, labels.ndim)
+    return sk_binary_erosion(labels, footprint=footprint)
+
+
+@register_function(menu="Segmentation post-processing > Grayscale dilation (scikit-image, nsbatwm)")
+@jupyter_displayable_output(library_name='nsbatwm', help_url='https://www.napari-hub.org/plugins/napari-segment-blobs-and-things-with-membranes')
+@time_slicer
+def grayscale_dilation(labels: "napari.types.LabelsData", radius: int = 1,
+                       footprint_shape: FootprintShape = FootprintShape.square) -> "napari.types.LabelsData":
```

Comment:
I'm not sure if the footprint_shape thingy is compatible with the Notebook generation.